include/revng/Pipeline/GenericLLVMPipe.h

```diff
@@ -133,6 +141,13 @@ class LLVMPassWrapper : public LLVMPassWrapperBase {
   std::unique_ptr<LLVMPassWrapperBase> clone() const override {
     return std::make_unique<LLVMPassWrapper>(*this);
   }
+
+  void print(llvm::raw_ostream &OS) const override {
+    if constexpr (LLVMPrintablePass<T>)
+      PipePass.print(OS);
+    else
+      OS << "-" << T::Name;
```

Comment:
First case is for custom LLVM pipes that might run multiple passes, while the second is for contractless passes.
Correct?

Reply:
yes and the first one is for those that have custom arguments to be set as well.